pycm/pycm_param.py

```diff
@@ -46,7 +46,14 @@
 </html>
 '''
 
-
+PROBABILITY_SIZE_ERROR = "Probability vector elements must have same length and equal to classes"
+PROBABILITY_TYPE_ERROR = "The elements of the probability vector can only contain numeric values"
+PROBABILITY_SUM_ERROR = "The sum of probability values must be one"
+THRESHOLDS_NUMBER_ERROR = "Number of the thresholds is lower than 2"
+THRESHOLDS_TYPE_ERROR = "The type of thresholds is assumed to be a list or a NumPy array"
```

Comment:
`Thresholds is assumed to be a list or a NumPy array`
or
`The type of thresholds is assumed to be list or NumPy array`, right?

Reply:
fixed in https://github.com/sepandhaghighi/pycm/pull/457/commits/8ad5728e2b9fba469d32c977d7be7e2a6e77d05f